src/sentry/static/sentry/app/bootstrap.tsx

```diff
@@ -70,9 +68,14 @@ const config = ConfigStore.getConfig();
 
 const tracesSampleRate = config ? config.apmSampling : 0;
 
+const hasReplays =
+  window.__SENTRY__USER &&
+  window.__SENTRY__USER.isStaff &&
+  process.env.NODE_ENV === 'production';
```

Comment:
Let's make it opt out?

```suggestion
  !!process.env.DISABLE_RR_WEB;
```

Reply:
imo unrelated to this change, and im not convinced we should support this behavior because if everyone opts out its getting zero passive QA

Reply:
people generally stick to defaults